Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM docker:19.03.2
-LABEL "repository"="https://github.com/ferluisxd/create-npmrc"
-LABEL "maintainer"="Luis Vilca"
+FROM docker:19.03.2 
+# TODO: Change this to run something other than Docker in Docker
```

Comment:
nit: Agreed this could quite easily just execute the `entrypoint.sh` script inside a native ubuntu container spun up by the action. But out of scope for this and can be done transparently in the future.